docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash -xe
+ tail -f /var/log/*
```

Comment:
In /var/log path you will find a lot of different files so this tail will show kern.log, syslog, dmesg, auth.log and so on. In my opinion you can try to read only ingestion logs using "tail -f /var/log/sds/ingestion-*" or not tail any file.